src/test/java/alvioneurope/gsergeev/javagroup/clientlogimporter/ExecutedCommandTest.java

```diff
@@ -0,0 +1,33 @@
+package alvioneurope.gsergeev.javagroup.clientlogimporter;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kintaro18 on 19.11.2015.
+ */
+public class ExecutedCommandTest {
+    private ExecutedCommand testExecutedCommand1;
+    private ExecutedCommand testExecutedCommand2;
+    private ExecutedCommand testExecutedCommand3;
+    private ExecutedCommand testExecutedCommand4;
+
+    @Before
+    public void setUp(){
+        testExecutedCommand1 = new ExecutedCommand("2015-11-10 11:37:33.495 Debug: CommandRunner::runCommand start  RequestId  \"19216805118520151110083733000000\"  commandId  67  commandName  \"UserAuthorization\"  timeout  -1  requestLength  400");
+        testExecutedCommand2 = new ExecutedCommand("2015-11-10 11:37:41.044 Debug: CommandRunner::runCommand end  RequestId  \"19216805118520151110083740000004\"  commandId  560  commandName  \"OrgElementFlatGet\"  responseLength  359184  execTime  367 ");
+        testExecutedCommand3 = new ExecutedCommand("2015-11-10 11:37:35.666 Debug: void UserCache::poolUser() ..\\mango_crm\\core\\usercache.cpp 12 Pool user list");
+        testExecutedCommand4 = new ExecutedCommand("");
+    }
+
```

Comment:
I think we should to analyse several use cases:
1. Correct input string without executed command
2. Correct input string with executed command
3. Arbitrary string
4. Part of string with executed command. I mean the situation when log has been cut or broken due to various reasons.
For example:
4.1 CommandRunner::runCommand end  RequestId  \"19216805118520151110083740000004\"  commandId
4.2 CommandRunner::runCommand end  RequestId  \"19216805118520151110083740000004\"  commandId YYY
4.3 CommandRunner::runCommand end  RequestId  \"19216805118520151110083740000004\"  commandId  560  commandName  \"OrgElementFlatGet\"  responseLength  3591gshs 
etc.

Please write a separately unit test for this use cases